tests/UI/Component/Listing/ListingTest.php

```diff
@@ -103,26 +103,6 @@ public function test_render_ordered_listing() {
 		$this->assertEquals($expected, $html);
 	}
 
-	public function test_ordered_invalid_items() {
-		$f = $this->getListingFactory();
-
-		try{
-			$f->ordered("1");
-		}catch(Exception $e){
-			$this->assertEquals($e->getCode(), E_RECOVERABLE_ERROR);
-		}
-	}
-
-	public function test_descriptive_invalid_items1() {
-		$f = $this->getListingFactory();
-
-		try{
-			$f->descriptive("1","1");
-		}catch(Exception $e){
-			$this->assertEquals($e->getCode(), E_RECOVERABLE_ERROR);
-		}
-	}
-
```

Comment:
Due to your recommendations we type hint on array for lists here. This results in an error in PHP7 (maybe also older versions) if string is given due to the type hint on PHP level. I did regard this test as superfluous since it makes limited sense to tests php type hinting capability. 

In this commit this is mainly here, that the tests run again.